postgres/01-init-user.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username ${POSTGRES_USER} --dbname ${POSTGRES_DB} <<-EOSQL
```

Comment:
please add Postgis extension

Reply:
solved here https://github.com/geosolutions-it/docker-geoserver/pull/43/commits/329db3edae5f726910e2dc637206841b6e51af9a